app/assets/stylesheets/admin/attachment.scss

```diff
@@ -0,0 +1,56 @@
+// Place all the styles related to the admin/attachment controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
+.table-attachment {
+  width: 100%;
+  border-collapse: collapse;
+}
+
+td, th {
```

Comment:
do you know that this CSS change affecting ALL td/th styles on all pages?

Reply:
fixed